hail/python/hailtop/hailctl/batch/batch_cli_utils.py

```diff
@@ -2,11 +2,12 @@
 import yaml
 import aiohttp
 import csv
-import sys
 from typing import List, Dict, Callable
 import tabulate
+import io
 
-LoD = List[Dict[str, str]]
+TableData = List[Dict[str, str]]
+choices = ['json', 'yaml', 'csv', *tabulate.tabulate_formats]
```

Comment:
Let's give this a more descriptive name and capitalize it since it is global and runtime-constant, like TABLE_FORMAT_OPTIONS